sections/featured-blog.liquid

```diff
@@ -93,7 +192,13 @@
             </div>
           </article>
         </li>
+            {% endif %}
       {% endfor %}
+      
```

Comment:
His logic seems to be: 1) Go through the number of articles determined by the post limit and if one's tag[0] is featured 2) then loop through the  number of articles determined by the post limit and look for one that is featured and render it 3) loop through the  number of articles determined by the post limit and if the tag is not featured 4) render it.

It shouldn't take two for loops and two checks to render a featured article. Plus it's not even checking all of the articles, just three.